assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -95,7 +95,8 @@ systemd:
           --pod-manifest-path=/etc/kubernetes/manifests \
           --read-only-port=0 \
           --register-with-taints=$${NODE_TAINTS} \
-          --volume-plugin-dir=/var/lib/kubelet/volumeplugins
+          --volume-plugin-dir=/var/lib/kubelet/volumeplugins \
+          --kube-reserved=memory=500Mi
```

Comment:
Where does this value come from?

Reply:
The values are a snapshot of `kubectl top pods --sort-by=memory | sort -h -k3 -r` after deploying a cluster from `example` configuration. They should perhaps be configurable, as depending on the cluster size, they will be changing.

Those values are minimal, just to have something in place.

Should I add such comment in commit message?

Reply:
Yes, please do.

I'd like to get some values from production clusters maybe too... But we can update later, maybe?

Reply:
Wow, kubelet+docker was using 500mb?

Note `ssh`, etc. should go under `--system-reserved`, say [the docs](https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#system-reserved)

Reply:
Ah right. I'll use system-reserved then.

Reply:
Done. PTAL.